lib/view_model/exchange/exchange_trade_view_model.dart

```diff
@@ -72,6 +72,18 @@ abstract class ExchangeTradeViewModelBase with Store {
     ? '\n\n' + S.current.xrp_extra_info
     : '';
 
+  @computed
+  String get pendingTransactionFiatAmountValueFormatted =>
+      sendViewModel.balanceViewModel.disableFiat
+          ? '' : sendViewModel.pendingTransactionFiatAmount
+          + ' ' + sendViewModel.fiat.title;
+
+  @computed
+  String get pendingTransactionFeeFiatAmountFormatted =>
+      sendViewModel.balanceViewModel.disableFiat
```

Comment:
we can create a computed getter in the `sendViewModel` instead so it can be easily tracked